USAGE.md

```diff
@@ -20,11 +20,9 @@ export VAULT_K8S_NAMESPACE=<your namespace>
 ### 4.1. Minikube
 Run the following commands to enable ingress:
 ```bash
-
 minikube addons enable ingress
-
```

Comment:
I moved this part into a dedicated section below @Xelef2000 because it's the same for Kind/minikube